doc/languages-frameworks/r.section.md

```diff
@@ -94,6 +94,43 @@ Executing `nix-shell` will then drop you into an environment equivalent to the
 one above. If you need additional packages just add them to the list and
 re-enter the shell.
 
+## torch with GPU acceleration {#torch-gpu-accel}
+
+The `torch` package included in `nixpkgs` is based on the binary packages by the
+mlverse team. By default, the CPU version of the package is installed. To enable
+GPU acceleration, set `config.cudaSupport` to true when importing `nixpkgs` in a
+`shell.nix`. You will also need to use `nixglhost`. Here is an example of such a
+shell:
+
+```nix
+{ pkgs ? import <nixpkgs> {config.cudaSupport = true; }, lib ? pkgs.lib }:
+
+let
+ nixglhost-sources = pkgs.fetchFromGitHub {
+   owner = "numtide";
+   repo = "nix-gl-host";
+   rev = "main";
+   # Replace this with the hash Nix will complain about, TOFU style.
+   hash = "";
+ };
+
+ nixglhost = pkgs.callPackage "${nixglhost-sources}/default.nix" { };
+
+in pkgs.mkShell {
+  buildInputs = [
+    nixglhost
+    pkgs.rPackages.torch
+    pkgs.R
```

Comment:
Shouldn't this be an rWrapper.override instead?

Reply:
If we drop in a shell this is not strictly required, but would be needed if calling R non-interactively afaik

Reply:
I see. This confuses people, let's not advertise this and use rWrapper instead: https://mastodon.acm.org/@zwol@hackers.town/112745541961800402

Reply:
thanks, done with latest push!